apps/privacy-panel/test/marionette/tc_main_test.js

```diff
@@ -86,6 +86,18 @@ marionette('transparency control panels', function() {
     assert.ok(find(app.chubbyU + ' ~ ' + app.flyingP));
     assert.ok(find(app.flyingP + ' ~ ' + app.mightyD));
 
+    // use the search box to filter apps
+    subject.enterSearchMode();
+    subject.sendSearchKeys('i'); // our 3 fake apps have an 'i' in their name
+    assert.ok(find(app.chubbyU).displayed());
+    assert.ok(find(app.flyingP).displayed());
+    assert.ok(find(app.mightyD).displayed());
+    subject.sendSearchKeys('g'); // only 'Mighty Duck' has 'ig' in its name
+    assert.ok(!find(app.chubbyU).displayed());
+    assert.ok(!find(app.flyingP).displayed());
+    assert.ok(find(app.mightyD).displayed());
+    subject.leaveSearchMode();
+
```

Comment:
lets add an integration test for
* the implemented clear button
* selecting an app then coming back